packages/rs-drive/src/fee/op.rs

```diff
@@ -290,6 +291,36 @@ impl LowLevelDriveOperation {
         GroveDbOpBatch::from_operations(operations)
     }
 
+    /// Filters the groveDB ops from a list of operations and puts them in a `GroveDbOpBatch`.
+    pub fn grovedb_operations_batch_consume(
```

Comment:
I would name it `into_grovedb_operations_batch` https://rust-lang.github.io/api-guidelines/naming.html